src/Client.php

```diff
@@ -180,18 +198,13 @@ protected function getEndpoint(string $endpoint): string
         return $this->baseUri.$endpoint;
     }
 
-    protected function mergeHeaders(array $headers = []): array
+    /**
+     * @param array $headers
+     *
+     * @return array
+     */
+    protected function mergeHeaders(array $headers): array
```

Comment:
Any reason to remove the default value here? Other than it being kind of useless since looking at what calls it.

Reply:
Just because it's useless.